shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -0,0 +1,116 @@
+    os.makedirs(nifti_path)
+
+    # Create bids structure for data
+    subprocess.run(['dcm2bids_scaffold', '-o', nifti_path], check=True)
+
+    # Copy original dicom files into nifti_path/sourcedata
+    copy_tree(unsorted_dicom_dir, os.path.join(nifti_path, 'sourcedata'))
+
+    # Call the dcm2bids_helper
+    subprocess.run(['dcm2bids_helper', '-d', unsorted_dicom_dir, '-o', nifti_path], check=True)
+
+    # Check if the helper folder has been created
+    helper_path = os.path.join(nifti_path, 'tmp_dcm2bids', 'helper')
+    if not os.path.isdir(helper_path):
+        raise ValueError('dcm2bids_helper could not create directory helper')
+
+    # Make sure there is data in nifti_path / tmp_dcm2bids / helper
+    helper_file_list = os.listdir(helper_path)
+    if not helper_file_list:
+        raise ValueError('No data to process')
+
+    # Create list of acquisitions
+    acquisition_names = []
+    acquisition_numbers = []
+    modality = []
+
+    # Create list containing all files
+    for iFile in range(len(helper_file_list)):
+        # If it's a json file
+        name, ext = os.path.splitext(helper_file_list[iFile])
+        if ext == '.json':
+            # Check for both .nii.gz and .nii
+            if name + '.nii.gz' in helper_file_list:
+                nifti_index = helper_file_list.index(name + '.nii.gz')
+            elif name + '.nii' in helper_file_list:
+                nifti_index = helper_file_list.index(name + '.nii')
+            else:
+                raise TypeError('Nifti file "' + name + '" not found')
+
+            nifti_file = helper_file_list[nifti_index]
+            # Read json file
+            _, _, json_data = read_nii(os.path.join(helper_path, nifti_file))
+
+            # Create future folder name
+            acquisition_numbers.append(json_data['SeriesNumber'])
+            acquisition_names.append(json_data['SeriesDescription'])
+            # Modality could be used as acquisition name
+            modality.append(json_data['Modality'])
+
+    # Remove duplicates
+    acquisition_numbers, ia = np.unique(acquisition_numbers, return_index=True)
+    acquisition_names_short = []
+    modality_short = []
+    for iAcq in ia:
+        acquisition_names_short.append(acquisition_names[iAcq])
+        modality_short.append(modality[iAcq])
+
+    # Folder where the different nifti acquisitions will be stored
+    output_dir = os.path.join(nifti_path, 'code')
+
+    for iAcq in range(len(acquisition_names_short)):
+        # Create a config.json file, place it in nifti_path / code
+        config_file_path = create_config(output_dir, acquisition_numbers[iAcq], acquisition_names_short[iAcq], modality_short[iAcq])
+
+        # Call dcm2bids
+        participant = ''
+        subprocess.run(['dcm2bids', '-d', unsorted_dicom_dir, '-o', nifti_path, '-p', participant, '-c', config_file_path], check=True)
+
+
+def create_config(output_dir, acquisition_number, acquisition_name, modality):
```

Comment:
what is the purpose of that function? creating docstrings right away helps the team understand the direction this code is taking